library/core/src/macros/mod.rs

```diff
@@ -6,15 +6,12 @@ macro_rules! panic {
     () => (
         $crate::panic!("explicit panic")
     );
-    ($msg:literal) => (
+    ($msg:literal $(,)?) => (
```

Comment:
There is no `($msg:literal,) => ($panic!($msg))` in the previous code, but this seems previously covered by `($msg:expr,) => ($panic!($msg))`.

Reply:
Ah, that's what confused me then.